pkg/config/config.go

```diff
@@ -224,6 +224,7 @@ func (c *Config) RegisterFlags(f *flag.FlagSet) {
 		"basic auth username for fetching remote config. (requires remote-configs experiment to be enabled")
 	f.StringVar(&c.BasicAuthPassFile, "config.url.basic-auth-password-file", "",
 		"path to file containing basic auth password for fetching remote config. (requires remote-configs experiment to be enabled")
+	f.StringVar(&c.AgentManagement.CacheLocation, "agentmanagement.cache-location", "", "Path to writeable directory where remote configs will be cached to disk")
```

Comment:
let's specify a default here? for example, `-metrics.wal-directory` defaults to `data-agent/`, so it is probably a good default

Reply:
@spartan0x117 I tried to run this via config flags but it is not working for me right now, getting:

```
2023/01/17 15:58:54 error loading config file remote-config.yaml: invalid initial config: path to cache must be specified in 'remote_config_cache_location'
```